src/monodroid/jni/dylib-mono.h

```diff
@@ -109,6 +109,36 @@ typedef struct {
 	const unsigned int size;
 } MonoBundledAssembly;
 
+// Keep this in sync with MonoObject in object.h
+typedef struct {
+	// MonoObject 2 pointers
+	void *_pad_mono_object[2];
+} MonoObject;
```

Comment:
Just ease of use in recreating the other structures with the proper padding. As mentioned though the idea would be to avoid this in fine.